### DISCUSSION TITLE
Create New Labels

### DISCUSSION THREAD

**Discussion by rob-secondstage** [community/community#4981] [Mobile] [answered]
I'd love the iOS mobile app to allow creating new labels.

**Comment by eliperkins** [comment, marked as answer]
Thanks for the feedback @rob-secondstage! Create a new label when one doesn't exist already is a handy feature that GitHub.com has already. I would love to see this in the mobile apps as well!

I've jotted your feedback down on an internal issue for this. I'll keep you posted!
